package framework.primarybackup;

import framework.Address;
import java.io.Serializable;
import lombok.Data;

@Data
class View implements Serializable {
    private final int viewNum;
    private final Address primary, backup;
}
